code/datums/hud.dm

```diff
@@ -106,6 +106,10 @@ GLOBAL_LIST_INIT(huds, list(
 /datum/atom_hud/proc/add_to_single_hud(mob/M, atom/A) //unsafe, no sanity apart from client
 	if(!M || !M.client || !A)
 		return
+	if (ismob(A))
+		var/mob/m = A
+		if (m.digitalinvis && isAI(M))

```

Comment:
I'm not too crazy about having both M & m in the same line. Could you change it to something else? lol

Reply:
> using uppercase local variable names